controllers/service.js

```diff
@@ -3,13 +3,13 @@ const statusCode = require('../constants/statusCode');
 const logger = require('../modules/logger');
 
 const { doesSuchServiceExist } = require('../services/validations/service');
-const { doesSuchSellerExist } = require('../services/validations/seller');
+const { doesSuchUserExist } = require('../services/validations/user');
 
 module.exports = {
 
   createService: (params) => {
     return new Promise((resolve, reject) => {
-      doesSuchSellerExist(params.sellerEmail)
+      doesSuchUserExist(params.sellerEmail)
```

Comment:
`params.sellerEmail`

Reply:
Yeah i missed it. i have tested APIs login, signup and profile update because they have been integrated in app. Services get/create/update APIs can contain other bugs too. We need to test those as well as refactor naming. Working on same (in separate PR)

Reply:
shall i make this change or let it be part of separate PR?

Reply:
Ideally, we should have this in this PR only.

Reply:
okay. another thought is we can create services of a seller so `seller` term is better than `user`. but then `doesSuchUserExist` might be confusing but we are not going to rename that. What do you think? refactor `sellerEmail` to `userEmail` or just `email`

Reply:
email